app/src/main/java/com/example/mvp/data/source/local/dao/UserDAO.kt

```diff
@@ -0,0 +1,9 @@
+package com.example.mvp.data.source.local.dao
+
+import com.example.mvp.data.model.User
+
+interface UserDAO {
+    fun addUser(vararg user: User) : Int
+    fun checkUser(userName: String) : Boolean
+    fun checkUser(userName: String, passWord: String) : Boolean
```

Comment:
Đặt tên hàm ý nghĩa hơn nhé 
```suggestion
    fun isExistUser(userName: String) : Boolean
    fun checkUser(userName: String, passWord: String) : Boolean
```